pkg/controllers/user/alert/deployer/upgradeimpl.go

```diff
@@ -132,6 +129,7 @@ func (l *alertService) Upgrade(currentVersion string) (string, error) {
 	}
 	newApp := app.DeepCopy()
 	newApp.Spec.ExternalID = newCatalogID
+	newApp.Spec.Answers["operator.enabled"] = "false"
```

Comment:
I don't like the way we only upgrade one of the answer value. Maybe we should apply all the answers we need in this version to the new app.